mmv1/products/certificatemanager/api.yaml

```diff
@@ -212,4 +212,207 @@ objects:
+      - !ruby/object:Api::Type::String
+        name: 'description'
+        description: |
+          A human-readable description of the resource.
+      - !ruby/object:Api::Type::String
+        name: 'createTime'
+        output: true
+        description: |
+          Creation timestamp of a Certificate Map. Timestamp is in RFC3339 UTC "Zulu" format,
+          accurate to nanoseconds with up to nine fractional digits.
+          Examples: "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
+      - !ruby/object:Api::Type::Time
+        name: 'updateTime'
+        description: |
+          Update timestamp of a Certificate Map. Timestamp is in RFC3339 UTC "Zulu" format,
+          accurate to nanoseconds with up to nine fractional digits.
+          Examples: "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
+        output: true
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: 'labels'
+        description: |
+          Set of labels associated with a Certificate Map resource.
+      - !ruby/object:Api::Type::Array
+        name: 'gclbTargets'
+        description: |
+          A list of target proxies that use this Certificate Map
+        output: true
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::Array
+              name: 'ipConfigs'
+              description: |
+                An IP configuration where this Certificate Map is serving
+              item_type: !ruby/object:Api::Type::NestedObject
+                properties:
+                  - !ruby/object:Api::Type::String
+                    name: 'ipAddress'
+                    description : |
+                      An external IP address
+                  - !ruby/object:Api::Type::Array
+                    name: 'ports'
+                    description : |
+                      A list of ports
+                    item_type: Api::Type::Integer
+            - !ruby/object:Api::Type::String
+              name: 'targetHttpsProxy'
+              description: |
+                Proxy name must be in the format projects/*/locations/*/targetHttpsProxies/*.
+                This field is part of a union field `target_proxy`: Only one of `targetHttpsProxy` or
+                `targetSslProxy` may be set.
+            - !ruby/object:Api::Type::String
+              name: 'targetSslProxy'
+              description: |
+                Proxy name must be in the format projects/*/locations/*/targetSslProxies/*.
+                This field is part of a union field `target_proxy`: Only one of `targetHttpsProxy` or
+                `targetSslProxy` may be set.
+  - !ruby/object:Api::Resource
+      name: 'CertificateMapEntry'
+      base_url: 'projects/{{project}}/locations/global/certificateMaps'
+      create_url: 'projects/{{project}}/locations/global/certificateMaps?certificateMapId={{name}}'
+      self_link: 'projects/{{project}}/locations/global/certificateMaps/{{name}}'
```

Comment:
I think you'll want to update these for the [`certificateMapEntries`](https://cloud.google.com/certificate-manager/docs/reference/rest/v1/projects.locations.certificateMaps.certificateMapEntries/get).